cli/tests/e2e/snapshots/test_baseline/test_match_based_key/error.txt

```diff
@@ -0,0 +1,6 @@
+Scanning 1 file.
+
+Some files were skipped or only partially analyzed.
```

Comment:
Is this line expected on a non-baseline scan?

Reply:
Yeah, this just means that there were files semgrep couldn't scan completely (partial parse error) or skipped for various reasons